### PR TITLE
Convert HTMLcov server gDoc to Markdown

### PR DESCRIPTION
**Pull request by brandonnsc**
Task #423

### CODE REVIEW

File: docs/Tools_htmlcov_server.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Proxy address to the s3 bucket
```
Comment (samarth9008) [inline docs/Tools_htmlcov_server.md:1]: It should be the name of the file `Tools - htmlcov server`

File: docs/Tools_htmlcov_server.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+<!-- toc -->
+
+- [Usage](#usage)
+- [Admin Configuration Info](#admin-configuration-info)
+  * [S3 bucket](#s3-bucket)
+  * [NGINX server](#nginx-server)
+
+<!-- tocstop -->
```
Comment (samarth9008) [inline docs/Tools_htmlcov_server.md:10]: The index should look like this
image

File: docs/Tools_htmlcov_server.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
```
Comment (samarth9008) [inline docs/Tools_htmlcov_server.md:11]: This line is missing

Proxy address to the s3 bucket

File: docs/Tools_htmlcov_server.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+3. Nesting example
+   - Upload at /htmlcov `s3://cryptokaizen-html/juraj_test2/CmTask8789`
+   - Access at
+     [http://172.30.2.44/juraj_test/CmTask8789](http://172.30.2.44/juraj_test/CmTask8789)
+
+## Admin Configuration Info
+
+### S3 bucket
+
+- Need to set bucket policy to only allow getObject operations from one IP:
```
Comment (samarth9008) [inline docs/Tools_htmlcov_server.md:30]: getObject -> `getObject`. Inside code format.

File: docs/Tools_htmlcov_server.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  (which is a NAT GATEWAY of our VPC)
+  - IMPORTANT STEP OTHERWISE THE SITE IS VISIBLE PUBLICLY
+  -
```
Comment (samarth9008) [inline docs/Tools_htmlcov_server.md:33]: Okay to remove this.

File: docs/Tools_htmlcov_server.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  - ![NGINX server](Tools_htmlcov_server_figs/image1.png)
+
+### NGINX server
+
+- [Juraj Smeriga](mailto:juraj.smeriga@simkoit.com) and
+  [saggese@gmail.com](mailto:saggese@gmail.com) have their public key stored on
+  the server in case access via ssh is needed (user: ubuntu)
+- Simple t3.nano instance (run ansible hardening playbook for additional
+  security)
+- Nginx installed with simple config added into /etc/nginx/sites-enabled/default
```
Comment (samarth9008) [inline docs/Tools_htmlcov_server.md:63]: `/nginx/sites-enabled/default`

File: docs/Tools_htmlcov_server.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
+# Tools - htmlcov server
```
Comment (DanilYachmenev) [inline docs/Tools_htmlcov_server.md:1]: Lets name it just `HTMLcov server`

update filename and README ref accordingly

File: docs/Tools_htmlcov_server.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+
+
```
Comment (DanilYachmenev) [inline docs/Tools_htmlcov_server.md:4]: lets leave just 1 empty line here. no need for 3

File: docs/Tools_htmlcov_server.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+- [Usage](#usage)
+- [Admin Configuration Info](#admin-configuration-info)
+  * [S3 bucket](#s3-bucket)
+  * [NGINX server](#nginx-server)
+
+<!-- tocstop -->
+
+## Proxy address to the s3 bucket
+
+[http://172.30.2.44](http://172.30.2.44) (available only behind VPN)
```
Comment (DanilYachmenev) [inline docs/Tools_htmlcov_server.md:18]: lets make it as a single point bullet list to comply with other docs style
`- [http://172.30.2.44](http://172.30.2.44) (available only behind VPN)`

File: docs/Tools_htmlcov_server.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+    ]
+  }
+  ```
+- Public access can be blocked:
+  - ![NGINX server](Tools_htmlcov_server_figs/image1.png)
+
+### NGINX server
+
+- [Juraj Smeriga](mailto:juraj.smeriga@simkoit.com) and
+  [saggese@gmail.com](mailto:saggese@gmail.com) have their public key stored on
```
Comment (DanilYachmenev) [inline docs/Tools_htmlcov_server.md:65]: lets refer to GP instead of `saggese@gmail.com`
so 
```
and [GP](mailto:saggese@gmail.com) 
```

**Review by DanilYachmenev — COMMENTED** [review]
looks good
see the nits

**Review by samarth9008 — APPROVED** [review]
LGTM

File: docs/Tools_htmlcov_server.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+      }
+    ]
+  }
+  ```
+- Public access can be blocked:
+  - ![NGINX server](Tools_htmlcov_server_figs/image1.png)
+
+### NGINX server
+
+- [Juraj Smeriga](mailto:juraj.smeriga@simkoit.com) and
```
Comment (gpsaggese) [inline docs/Tools_htmlcov_server.md:62]: Can you change it to `jsmeriga@crypto-kaizen.com`?

File: docs/Tools_htmlcov_server.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+  [GP](mailto:saggese@gmail.com) have their public key stored on
```
Comment (gpsaggese) [inline docs/Tools_htmlcov_server.md:63]: gp@crypto-kaizen.com

File: docs/Tools_htmlcov_server.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+
+## Proxy address to the s3 bucket
+
+- [http://172.30.2.44](http://172.30.2.44) (available only behind VPN)
+
+## Usage
+
+1. Upload the coverage into bucket: **cryptokaizen-html** (located on CK AWS
+   account), note the syntax from CLI:
+   - `aws s3 cp ./htmlcov s3://cryptokaizen-html/juraj_test --recursive --profile ck`
```
Comment (gpsaggese) [inline docs/Tools_htmlcov_server.md:22]:
```
> aws s3 ...
```

**Review by gpsaggese — COMMENTED** [review]
Minor thing and merge.

We can assign another issue since this is so close

File: docs/Tools_htmlcov_server.md (hunk 13 above)
Comment (brandonnsc) [reply docs/Tools_htmlcov_server.md:22]: Does this mean using `aws s3 ...` to replace `aws s3 cp ./htmlcov s3://cryptokaizen-html/juraj_test --recursive --profile ck`

**Review by DanilYachmenev — COMMENTED** [review]
some nits from my end and done

File: docs/Tools_htmlcov_server.md (hunk 13 above)
Comment (samarth9008) [reply docs/Tools_htmlcov_server.md]: No, make it a multi-line code format with triple backticks and also start your command with `>`

Comment (samarth9008) [reply docs/Tools_htmlcov_server.md]: Remove the bullet point also

**Comment by samarth9008** [comment]
Do not forget ro resolve the conflict since we have moved the readme file

**Comment by brandonnsc** [comment]
> Do not forget ro resolve the conflict since we have moved the readme file

@samarth9008 Would you mind informing me of the current location of the `readme` file? Is it still residing in the `docs` folder or has it been moved to a different one?

**Comment by samarth9008** [comment]
Its in the root of the repo. We have merged it over there.

**Comment by DanilYachmenev** [comment]
fix the conflict please

File: docs/README.md
Comment (DanilYachmenev) [inline docs/README.md:1]: oh this file was deprecated
remove it from your commit pls

**Review by DanilYachmenev — COMMENTED** [review]
lgtm
just remove the obsolete file and all done imo

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Pull request merged**